Jenkinsfile

```diff
@@ -1,26 +1,73 @@
 pipeline {
 	agent any
+
 	options {
-		buildDiscarder logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: '', daysToKeepStr: '', numToKeepStr: '5')
 		disableConcurrentBuilds()
 	}
+
 	stages {
-		stage('Say Hello') {
+		stage("Unit Tests") {
 			steps {
-				echo "Hello"
+				script {
+					try {
+						if (isUnix()) {
+							sh(script: "./gradlew clean test --console=plain --no-daemon")
+						}
+						else {
+							bat(script: "./gradlew clean test --console=plain --no-daemon")
```

Comment:
Would it make sense to run `gradlew.bat` here (and in all the other `bat` statements)?